src/MicrosoftTeams.schema.json

```diff
@@ -35,17 +35,20 @@
                 "websiteUrl": {
                     "type": "string",
                     "description": "The url to the page that provides support information for the app.",
-                    "maxLength": 2048
+                    "maxLength": 2048,
+                    "pattern": "^[Hh][Tt][Tt][Pp][Ss]*://"
```

Comment:
I think "?" is more appropriate than "*" since you want 0 or 1 rather than 0 to N instances of the letter "s" or "S" in there, i.e.:
"pattern": "^[Hh][Tt][Tt][Pp][Ss]?://"